composer.json

```diff
@@ -8,8 +8,11 @@
   "homepage": "https://github.com/Adyen/adyen-php-api-library",
   "license": "Apache-2.0",
   "require": {
-    "php": ">=5.3",
-    "monolog/monolog": ">=1.16"
+    "php": ">=5.4",
```

Comment:
in order to support the hex2bin() function

Reply:
This would be a breaking change so for now let's not move the function which requires this function :(